cmd/service_env_set.go

```diff
@@ -17,14 +19,26 @@ func (o *ServiceEnvSetOperation) Validate() {
 	}
 }
 
-func (o *ServiceEnvSetOperation) SetEnvVars(inputEnvVars []string) {
+func (o *ServiceEnvSetOperation) SetEnvVars(inputEnvVars []string, envVarFile string) {
+	if envVarFile != "" {
+		file, err := os.Open(envVarFile)
+		if err != nil {
+			return
+		}
+		defer file.Close()
+		scanner := bufio.NewScanner(file)
+		for scanner.Scan() {
+			inputEnvVars = append(inputEnvVars, scanner.Text())
+		}
+	}
 	o.EnvVars = extractEnvVars(inputEnvVars)
 }
 
 var flagServiceEnvSetEnvVars []string
+var flagServiceEnvSetEnvFile string
 
 var serviceEnvSetCmd = &cobra.Command{
-	Use:   "set --env <key=value> [--env <key=value] ...",
+	Use:   "set --env <key=value> [--env <key=value] [--file filename] ...",
 	Short: "Set environment variables",
 	Long: `Set environment variables
```

Comment:
Probably should also add the new file option to the `Long` property of serviceEnvSetCmd.